### ISSUE TITLE
Обновить «Сортировку»

### ISSUE THREAD

**Issue by Mercury13** [roginvs/space-rangers-quest#72]
Я уже упоминал глюк с пустыми путями. Исправили.

**Comment by roginvs** [comment]
Это в новых версиях КР2 ?

**Comment by Mercury13** [comment]
Да, «Сортировку» и «Тюрьму-2» обновили в этот Новый год. Когда «Киберразум», не знаю.

**Comment by Mercury13** [comment]
Точно знаю, что в «Сортировке» ошибки, и не ваши — разговаривал ведь с вами. Про остальные не знаю.

**Comment by roginvs** [comment]
Напиши мне на почту, обновим
